[rospy.client][INFO] 2022-03-17 16:28:27,010: init_node, name[/talker], pid[20213]
[xmlrpc][INFO] 2022-03-17 16:28:27,011: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2022-03-17 16:28:27,012: Started XML-RPC server [http://ubuntu:45985/]
[rospy.init][INFO] 2022-03-17 16:28:27,013: ROS Slave URI: [http://ubuntu:45985/]
[rospy.impl.masterslave][INFO] 2022-03-17 16:28:27,014: _ready: http://ubuntu:45985/
[rospy.registration][INFO] 2022-03-17 16:28:27,016: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2022-03-17 16:28:27,016: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2022-03-17 16:28:27,017: Registering publisher topic [/video/click] type [geometry_msgs/PointStamped] with master
[rospy.internal][INFO] 2022-03-17 16:28:27,104: topic[/video/click] adding connection to [/listener], count 0
[rospy.init][INFO] 2022-03-17 16:28:27,114: registered with master
[rospy.rosout][INFO] 2022-03-17 16:28:27,115: initializing /rosout core topic
[rospy.rosout][INFO] 2022-03-17 16:28:27,118: connected to core topic /rosout
[rospy.simtime][INFO] 2022-03-17 16:28:27,121: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2022-03-17 16:28:27,349: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2022-03-17 16:28:28,734: topic[/video/click] adding connection to [/link_ros], count 1
[rospy.internal][INFO] 2022-03-17 20:32:38,620: topic[/video/click] removing connection to /link_ros
[rosout][WARNING] 2022-03-17 20:32:39,604: Inbound TCP/IP connection failed: connection from sender terminated before handshake header received. 0 bytes were received. Please check sender for additional details.
[rospy.internal][ERROR] 2022-03-17 20:32:39,609: Inbound TCP/IP connection failed:
Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 325, in _tcp_server_callback
    header = read_ros_handshake_header(sock, StringIO(), buff_size)
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 359, in read_ros_handshake_header
    raise ROSHandshakeException("connection from sender terminated before handshake header received. %s bytes were received. Please check sender for additional details."%b.tell())
ROSHandshakeException: connection from sender terminated before handshake header received. 0 bytes were received. Please check sender for additional details.

[rospy.internal][INFO] 2022-03-17 20:32:51,860: topic[/video/click] adding connection to [/link_ros], count 1
[rospy.internal][INFO] 2022-03-17 23:01:43,931: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2022-03-17 23:01:45,934: topic[/video/click] removing connection to /link_ros
[rospy.rosout][ERROR] 2022-03-17 23:01:46,003: Unable to report rosout: [Errno 111] Connection refused
Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/rosout.py", line 91, in _rosout
    disable_topics_ = rospy.get_param_cached("/rosout_disable_topics_generation", False)
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/client.py", line 490, in get_param_cached
    return _param_server.get_param_cached(param_name)
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/msproxy.py", line 168, in get_param_cached
    code, msg, value = self.target.subscribeParam(rospy.names.get_caller_id(), rospy.core.get_node_uri(), resolved_key)
  File "/usr/lib/python2.7/xmlrpclib.py", line 1243, in __call__
    return self.__send(self.__name, args)
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/core.py", line 670, in _ServerProxy__request
    self, methodname, params)
  File "/usr/lib/python2.7/xmlrpclib.py", line 1602, in __request
    verbose=self.__verbose
  File "/usr/lib/python2.7/xmlrpclib.py", line 1283, in request
    return self.single_request(host, handler, request_body, verbose)
  File "/usr/lib/python2.7/xmlrpclib.py", line 1311, in single_request
    self.send_content(h, request_body)
  File "/usr/lib/python2.7/xmlrpclib.py", line 1459, in send_content
    connection.endheaders(request_body)
  File "/usr/lib/python2.7/httplib.py", line 1095, in endheaders
    self._send_output(message_body)
  File "/usr/lib/python2.7/httplib.py", line 898, in _send_output
    self.send(msg)
  File "/usr/lib/python2.7/httplib.py", line 860, in send
    self.connect()
  File "/usr/lib/python2.7/httplib.py", line 837, in connect
    self.timeout, self.source_address)
  File "/usr/lib/python2.7/socket.py", line 575, in create_connection
    raise err
error: [Errno 111] Connection refused

[rosout][WARNING] 2022-03-17 23:01:45,997: Inbound TCP/IP connection failed: connection from sender terminated before handshake header received. 0 bytes were received. Please check sender for additional details.
[rospy.internal][ERROR] 2022-03-17 23:01:46,004: Inbound TCP/IP connection failed:
Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 325, in _tcp_server_callback
    header = read_ros_handshake_header(sock, StringIO(), buff_size)
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 359, in read_ros_handshake_header
    raise ROSHandshakeException("connection from sender terminated before handshake header received. %s bytes were received. Please check sender for additional details."%b.tell())
ROSHandshakeException: connection from sender terminated before handshake header received. 0 bytes were received. Please check sender for additional details.

[rospy.core][INFO] 2022-03-17 23:17:56,223: signal_shutdown [signal-2]
[rospy.registration][WARNING] 2022-03-17 23:17:56,225: cannot unregister with master due to network issues
[rospy.internal][INFO] 2022-03-17 23:17:56,226: topic[/video/click] removing connection to /listener
[rospy.impl.masterslave][INFO] 2022-03-17 23:17:56,227: signal-2
[rospy.core][INFO] 2022-03-17 23:17:56,393: signal_shutdown [atexit]
